src/main/java/dev/galacticraft/mod/api/block/PipeShapedBlock.java

```diff
@@ -109,10 +131,21 @@ public void neighborChanged(BlockState state, Level level, BlockPos pos, Block b
 
     @Override
     protected @NotNull VoxelShape getShape(BlockState state, BlockGetter world, BlockPos pos, CollisionContext context) {
-        if (world.getBlockEntity(pos) instanceof Connected connected) {
-            return this.shapes[generateAABBIndex(connected)];
+        VoxelShape baseShape = Shapes.empty();
```

Comment:
Although this theoretically shouldn't ever occur, it's probably better to fallback to `this.shapes[0]` (as before) rather than empty.